ext/mxnet/mxnet_internal.h

```diff
@@ -230,9 +230,10 @@ struct mxnet_api_table {
                           const char *key,
                           const char **out,
                           int *success);
-  int (* MXSymbolSetAttr)(SymbolHandle symbol,
-                          const char *key,
-                          const char *value);
+  int (* NNSymbolSetAttrs)(SymbolHandle symbol,
+                           mx_uint num_attrs,
```

Comment:
I want to add and use `nn_uint` like `mx_uint`.

Reply:
fixed in c5227f7